tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Responsive CV using HTML CSS</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
```

Comment:
Its a best practice to link our styles after the ones that you've imported from the external source.

Reply:
I will fix it